apps/server/src/modules/task/controller/api-test/task-copy-timeout.api.spec.ts

```diff
@@ -15,6 +14,7 @@ import {
 	taskFactory,
 	userFactory,
 } from '@shared/testing';
+import { JwtAuthGuard } from '@src/infra/auth-guard/guard/jwt-auth.guard';
```

Comment:
```suggestion
import { JwtAuthGuard } from '@infra/auth-guard';
```
Shouldn't it be coming from here?